webpack.plugins.js

```diff
@@ -0,0 +1,9 @@
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin')
+const VueLoaderPlugin = require('vue-loader/lib/plugin')
+
+module.exports = [
+  new ForkTsCheckerWebpackPlugin(),
```

Comment:
Why are these two modules loaded within a different file, and not in the main `plugins`-property?

Reply:
Tobi: include in renderer

Reply:
Done